Nef_S2/include/CGAL/Nef_S2/Sphere_circle.h

```diff
@@ -78,15 +80,20 @@ create any great circle that contains $p$ and $q$.}*/
   }
 }
 
- Sphere_circle(const Plane_3& h) : Base(h)
+Sphere_circle(const Plane_3& h) : Base(h)
 /*{\Mcreate creates the circle of $S_2$ corresponding to the plane
 |h|. If |h| does not contain the origin, then |\Mvar| becomes the
 circle parallel to |h| containing the origin.}*/
 {
-  if(h.d() != 0) *this = Plane_3(h.a(),h.b(),h.c(),RT(0));
+  if(h.d() != 0)
+    *this = Plane_3(h.a(),h.b(),h.c(),RT(0));
 }
 
-Sphere_circle(const RT& x, const RT& y, const RT& z): Base(x,y,z,0) {}
+Sphere_circle(const Origin& o, const Vector_3& v) : Base(o,v) {}
+
+Sphere_circle(const Origin&, const Plane_3& h) : Base(h.a(),h.b(),h.c(),RT(0)) {}
```

Comment:
We should add an assertion that the plane goes through the origin or at least put that in a comment.

Reply:
In this constructor, it assumes that the given plane does not pass through the origin, and sets d explicitly to 0. So I think a comment is appropriate. I will add one.